### PR TITLE
 Add get_vertex implementation & basic test for delta loader db 

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

**Comment by MrCreosote** [comment]
Yeah, we can just reimplement the DB methods if we need to

File: relation_engine/database/test/batch_time_travel_test.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    sys.delete_database(DB_NAME)
+
+def test_get_vertex(arango_db):
+    col_name = 'verts'
+    arango_db.create_collection(col_name)
+    col = arango_db.collection(col_name)
+
+    # it's assumed that given an id and a timestamp there's <= 1 match in the collection
+    col.import_bulk([{'_key': '1', 'id': 'foo', 'created': 100, 'expires': 600},
```
Comment (jayrbolton) [inline relation_engine/database/test/batch_time_travel_test.py]: Test for get_vertex on 'bar' at 100? Throws an error right?

File: relation_engine/database/test/batch_time_travel_test.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    ret = att.get_vertex('bar', 300)
+
+    assert ret == {'_key': '3', 'id': 'bar', 'created': 201, 'expires': 300}
+
+    ret = att.get_vertex('foo', 250)
+
+    assert ret == {'_key': '1', 'id': 'foo', 'created': 100, 'expires': 600}
+
+    assert att.get_vertex('bar', 50) == None
```
Comment (jayrbolton) [inline relation_engine/database/test/batch_time_travel_test.py]: And a test for 'bar' at 601 should be None

**Comment by jayrbolton** [comment]
Added a couple minor test comments

File: relation_engine/database/test/batch_time_travel_test.py (hunk 1 above)
Comment (MrCreosote) [reply relation_engine/database/test/batch_time_travel_test.py]: That'll get the node, the range is inclusive. 99 would return `None`.

Comment (jayrbolton) [reply relation_engine/database/test/batch_time_travel_test.py]: Sorry, me type bad. I meant to say to add another 'bar' active at 100 to test that the error is thrown

File: relation_engine/database/test/batch_time_travel_test.py (hunk 2 above)
Comment (MrCreosote) [reply relation_engine/database/test/batch_time_travel_test.py]: Added a test at 401

File: relation_engine/database/test/batch_time_travel_test.py (hunk 1 above)
Comment (MrCreosote) [reply relation_engine/database/test/batch_time_travel_test.py]: done, although used 200

**Comment by MrCreosote** [comment]
@jayrbolton we good?

**Comment by MrCreosote** [comment]
Quoting @jayrbolton:
> we good

**Pull request merged**